ui-lib/src/main/java/cash/z/ecc/ui/screen/home/viewmodel/WalletViewModel.kt

```diff
@@ -92,13 +92,15 @@ class WalletViewModel(application: Application) : AndroidViewModel(application)
         .filterIsInstance<SecretState.Ready>()
         .flatMapConcat {
             callbackFlow {
-                val synchronizer = SynchronizerCompanion.load(application, it.persistableWallet)
+                synchronizerMutex.withLock {
+                    val synchronizer = SynchronizerCompanion.load(application, it.persistableWallet)
```

Comment:
would this Mutex with lock stuff needed in every client using the SDK? if so, is it documented in the SDK and in the Demo App?

Reply:
It shouldn't block clients of the SDK.  However, looking at it again, this isn't doing what I intended.

I was trying to avoid issues with the ordering of stop/start calls, because the wipe operation needs to stop/start.  Unfortunately, I can't use the Mutex inside `awaitClose` because that block isn't in a coroutine context.

I just pushed another change which should make that better.

Reply:
Make that 2 changes 😬